dora/core/server/worker/src/main/java/alluxio/worker/http/HttpServer.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.http;
+
+import com.google.common.base.Throwables;
+import com.google.inject.Inject;
+import io.netty.bootstrap.ServerBootstrap;
+import io.netty.channel.Channel;
+import io.netty.channel.ChannelOption;
+import io.netty.channel.EventLoopGroup;
+import io.netty.channel.nio.NioEventLoopGroup;
+import io.netty.channel.socket.nio.NioServerSocketChannel;
+import io.netty.handler.logging.LogLevel;
+import io.netty.handler.logging.LoggingHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * {@link HttpServer} provides Alluxio RESTful API. It is implemented through Netty.
+ */
+public final class HttpServer {
+
+  private static final Logger LOG = LoggerFactory.getLogger(HttpServer.class);
+  static final boolean SSL = false;
+  static final int PORT = 28080;
+
+  private final HttpServerInitializer mHttpServerInitializer;
+
+  /**
+   * {@link HttpServer} provides Alluxio RESTful API. It is implemented through Netty.
+   * @param httpServerInitializer this object initializes the Netty pipeline of HTTP Server
+   */
+  @Inject
+  public HttpServer(HttpServerInitializer httpServerInitializer) {
+    mHttpServerInitializer = httpServerInitializer;
+  }
+
+  /**
+   * Starts the HTTP server.
+   */
+  public void start() {
+    // Configure the server.
+    EventLoopGroup bossGroup = new NioEventLoopGroup(1);
```

Comment:
I remember we have a discussion about this before. the HTTP server would share the same event loop group with the data server. This can better control resources.
the http server and data server can be in the same ServerBootstrap, just in two pipelines. what do you think of it?

Reply:
When I tried to use the same ServerBootstrap, I found that the implementation would be very complicated as you mentioned before. And the port of HTTP server must be the same with the existing one. For fast testing, I just create a new one here.

Reply:
ok, for fast testing, it's not a big deal. I will take a try in the s3 api.

Reply:
> ok, for fast testing, it's not a big deal. I will take a try in the s3 api.

Feel free to modify the code if needed.